qadence_protocols/types.py

```diff
@@ -25,3 +25,17 @@ class Protocols(StrEnum):
 
 
 qadence_available_protocols = Protocols.list()
+
+
+class ReadOutOptimization(StrEnum):
+    # basic inversion and maximum likelihood estimate
+    MLE = "mle"
+
```

Comment:
Remove.

Reply:
I am capitalizing it, not sure exactly what you want me to remove here

Reply:
Ah the blank line. Sorry, been unclear.